### ISSUE TITLE
Train.py says it's finished training after 1 second.

### ISSUE THREAD

**Issue by me990th** [daniel-kukiela/nmt-chatbot#171]
I ran all the setup files, but when I run Train.py it only takes 1 second, and doesn't train.   This is what it says:

[32mTraining model...[39m

[32mEpoch: 1, learning rate: 0.001 - already passed[39m
[32mEpoch: 2, learning rate: 0.0001 - already passed[39m
[32mEpoch: 3, learning rate: 1e-05 - already passed[39m

[32mTraining finished[39m

Any help would be nice.

**Comment by Scum1254** [comment]
i think youleft the degsult settings for how much epochs you want to train.
thats only three epochs on default. long time ago when i was workung wirh
this project bit i think there is a line where ou can add more epochs.
Sonething like epochs: „0.001“, „0.00001“ ecet.

me990th <notifications@github.com> schrieb am Di. 22. Dez. 2020 um 13:26:

**Comment by NoCodeAvaible** [comment]
Do you got a soloution for your Issue?
